### DISCUSSION TITLE
freeform.js incomplete in webpage and form submissions all get status Spam

### DISCUSSION THREAD

**Discussion by g-ratsch** [solspace/craft-freeform#900] [Questions & Answers]
**Describe the bug or issue you're experiencing**
- All freeform submissions are marked as Spam
- In the browser we get the error Unexpected token ')' 

**Steps to reproduce**
We used {{ form.renderClosingTag }} to close the form (build in Twig).

In the browser we noticed that parts of the code in freeform.js is gone:
 original -------------------------------Found in the website
 {method:e,url:t,data:(r||{}).data} -------- {).data}
 ({target:t,proto:!0,forced:m||j},_) ------- (j,_
This is tested with Chrome and Edge browser.

Because of this the freeform js file is not used.

**Expected behavior**
I have replaced {{ form.renderClosingTag } with  \</form\> to close the form .
And In de 'body' i added {{ craft.freeform.loadFreeformPlugin() }}

**Result**: the complete js is added to the webpage and the Honeypot is working again.
 
**Craft & Plugin Info (please complete the following information):**

- Craft Version: 3.8
- Freeform Version: 3.13.29
- Freeform Edition:
- Fresh Install or Upgrade: Upgrade

**Additional context**
It works for me at this moment. It is not the best solution because the freeform js is now loaded on every page.

**Comment by kjmartens** [comment]
Sorry for the trouble you're experiencing, @g-ratsch.

I'm curious about what your complete template code looks like. And can you send or attach a screenshot of the Freeform Diagnostics page? 🙂

**Reply by g-ratsch** [reply]
Sorry for the late reaction.
I will try to anwser your questions this week. 
I have to look what 'Freeform Diagnotics page' is and how it works.

with regards
Gerd Ratsch
